transport/src/androidUnitTest/kotlin/transport/core/MessageExtensionTest.kt

```diff
@@ -171,7 +172,7 @@ internal class MessageExtensionTest {
             assertThat(id).isEqualTo(expectedMessage.id)
             assertThat(direction).isEqualTo(expectedMessage.direction)
             assertThat(state).isEqualTo(expectedMessage.state)
-            assertThat(type).isEqualTo(expectedMessage.type)
+            assertThat(messageType).isEqualTo(expectedMessage.messageType)
```

Comment:
lets keep both validations for now. At least, until we remove the derpecated `type` fieldl from Message object